fenix/app/onboarding.fml.yaml

```diff
@@ -25,8 +25,7 @@ features:
               card-type: default-browser
               title: juno_onboarding_default_browser_title_nimbus_2
               ordering: 10
-              body: juno_onboarding_default_browser_description_nimbus_2
-              link-text: juno_onboarding_default_browser_description_link_text
```

Comment:
If we are going to remove this and only use the app string for caption, I think we should also remove this field from the OnboardingCardData type in this file.

Reply:
Agreed, I'll do that.  Thanks